components/builder-originsrv/src/data_store.rs

```diff
@@ -775,8 +845,10 @@ impl DataStore {
     ) -> SrvResult<originsrv::OriginPackage> {
         let conn = self.pool.get(opc)?;
         let ident = opc.get_ident();
+        let visibility = format!("{}", &opc.get_visibility());
```

Comment:
Can we implement `ToString` for visibilities instead of this?

Reply:
Turns out you get `ToString` for free when you implement `Display`, which we've already done.  I didn't realize that.  I'll update these calls.

Reply:
Ah, right 😄